### ISSUE TITLE
The `outline-none` rule should catch more cases

### ISSUE THREAD

**Issue by EvgenyOrekhov** [CSSLint/csslint#723]
The `outline-none` rule should generate a warning for the following case:

```css
:focus {
    outline: transparent;
}
```

Also, it should generate warnings for any values that use either `0`, `none`, or `transparent`, e. g.

```css
:focus {
    outline: 1px none;
}
:focus {
    outline: 0 red;
}
:focus {
    outline: 1px solid transparent;
}
// and other combinations and permutations
```

**Comment by EvgenyOrekhov** [comment]
I am no longer interested in this. Feel free to reopen if needed.
